src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx

```diff
@@ -55,37 +55,105 @@ export const ActivityGroupAmountChange: FC<Props> = ({ nonZeroAmounts, textSize
       );
     }
 
-    if (isDoubleTokenOperation(nonZeroAmounts)) {
-      const symbols = nonZeroAmounts.map(({ symbol }) => symbol).join(',');
+    const { positiveAmounts, negativeAmounts } = separateAmountsBySign(nonZeroAmounts);
+
+    if (positiveAmounts.length === 0) {
```

Comment:
Please rewrite it to avoid nested conditionals. Using `switch`...`case` is encouraged here.